### DISCUSSION TITLE
Konfiguration Leistung WW

### DISCUSSION THREAD

**Discussion by RaikertHA** [Trunks1982/Daikin-Rotex-HPSU-CAN#44] [Ideas]
Hallo Trunks1982,

danke für die tolle Arbeit, um die alte Rotex ins Samrt Home zu integrieren - funktioniert top!

Eine Entwicklungsanfrage:
Wäre es möglich, die Leistung WW konfigurierbar zu machen (0, 3 ,6, 9 kW), auch wenn diese Einstellung im "geschützten Konfigurationsbereich" der ROTEX liegt?

Hintergrund:
Bei stark schwankender Leistung der PV-Anlage wäre es sehr hilfreich, wenn die Leistung des Backup-Heaters je nach verfügbarer PV-Überschussleistung angepasst - also quasi nachgeführt - werden könnte. Somit kann man die jeweils maximal verfügbare Überschussleistung ins Warmwasser speichern. 

Ich hoffe die Idee ist verständlich dargestellt worden.

Vielen Dank vorab für ein Feedback.

Viele Grüße
RaikertHA

**Comment by Trunks1982** [comment]
Möglich ist es.
Nutzt du es mit Smart Grid zusammen?

**Comment by RaikertHA** [comment]
Ja genau mit SG zusammen.Am 28.06.2024 15:24 schrieb Trunks1982 ***@***.***>:
Möglich ist es.
Nutzt du es mit Smart Grid zusammen?
